#occupancyLengths:
#    r1: 4
#    r2: 4

lockingOff: false

limitedAreas:
    - area:
          nodes: [71, 78, 79, 80, 82, 84, 86]
          maxNumClients: 1
